modules/ROOT/pages/provisioning-exoscale.adoc

```diff
@@ -0,0 +1,64 @@
+= Provisioning Fedora CoreOS on Exoscale
+
+This guide shows how to provision new Fedora CoreOS (FCOS) instances on https://exoscale.com[Exoscale] Cloud Hosting.
+
+== Prerequisites
+
+Before provisioning a FCOS instance, you can use or not an Ignition configuration file containing your customizations. If you need one, see xref:producing-ign.adoc[Producing an Ignition File].
+
+You also need to have access to an Exoscale account. https://portal.exoscale.com/register[Register] if you don't have one.
+
+== About images
+
+Fedora CoreOS is updated at every major change on our platform. The future goal is to have Exoscale FCOS in https://getfedora.org/en/coreos/download?tab=cloud_launchable[Cloud Lauchable] section, to be updated at every release.
```

Comment:
I'd rewrite the first sentence as:
"Fedora CoreOS image are periodically update by Exoscale on their platform. The future ..."

Reply:
1c31a1f